recipe/meta.yaml

```diff
@@ -9,24 +9,26 @@ source:
   sha256: 52a0e2cd5fc2255ab7c8a91f874148b177b2ce9032f43b3f4844bbc27ccffd38
 
 build:
-  number: 0
-  skip: true  # [py < 37]
+  number: 1
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - {{ compiler('cxx') }}
     - {{ cdt('mesa-libgl-devel') }}      # [linux]
     - {{ cdt('libxi-devel') }}           # [linux]
     - ninja
     - cmake
-    - swig >=3.0.11
+    - swig
   host:
     - python
-    - occt
+    - occt {{ version }}
     - tbb-devel
   run:
     - python
-    - occt
+    - occt                  # [unix]
+    - {{ pin_compatible('occt', max_pin='x.x.x') }}  # [win]
```

Comment:
I guess we should use this for all platforms. occt must be tightly pinned. But it seems like this syntax doesn't work (looking at the latest windows builds)

Reply:
Okay I've done the run_constrained (which works) for all platforms in the latest commit. No idea why the pin_compatible doesn't work.